rust/signer/src/signer.udl

```diff
@@ -492,7 +492,7 @@ interface ScreenData {
     Settings(MSettings f);
     Log(MLog f);
     LogDetails(MLogDetails f);
-    Transaction(MTransaction f);
+    Transaction(sequence<MTransaction> f);
```

Comment:
Any type transactions? Or we can any multiple "signature" transactions or any single one?
If second I would have separate data model for it.